src/lib/features/playground/offline-unleash-client.ts

```diff
@@ -64,7 +64,7 @@ export const offlineUnleashClient = async ({
         },
     });
 
-    client.start();
+    await client.start();
```

Comment:
start is async so await is needed. w/o it the code would break if we switched to node.js sdk